### PR TITLE
Add Jimmy Song's nomination for Steering Committee elections

### PR DESCRIPTION
**Pull request by wu-sheng**
Follow https://github.com/istio/community/issues/428#issuecomment-687138580

### CODE REVIEW

File: steering/elections/2020/jimmysong.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Jimmy Song
+
+- GitHub: https://github.com/rootsongjc
+- Affiliation: developer advocate, Tetrate.io
+- Slack: @Jimmy Song
+- Twitter: @jimmysongio
+- LinkedIn: https://www.linkedin.com/in/jimmysongio/
+
+# Bio
+Jimmy is a developer advocate at Tetrate, CNCF ambassador,
```
Comment (linsun) [inline steering/elections/2020/jimmysong.md:10]: Isn't Jimmy also one of the key maintainers for istio.io chinese website?

Comment (wu-sheng) [reply steering/elections/2020/jimmysong.md:10]: He wrote the bio for me :) @rootsongjc anything to update?

Comment (rootsongjc) [reply steering/elections/2020/jimmysong.md:10]: Yes, I am the initiator and key maintainer of the Istio Chinese website.

File: steering/elections/2020/jimmysong.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- GitHub: https://github.com/rootsongjc
+- Affiliation: developer advocate, Tetrate.io
+- Slack: @Jimmy Song
+- Twitter: @jimmysongio
+- LinkedIn: https://www.linkedin.com/in/jimmysongio/
+
+# Bio
+Jimmy is a developer advocate at Tetrate, CNCF ambassador, 
+co-founder of ServiceMesher Community which is the largest Service Mesh Community of the world with 4000+ members.
```
Comment (rootsongjc) [inline steering/elections/2020/jimmysong.md:11]:
```suggestion
co-founder of ServiceMesher Community which is the largest Service Mesh Community of the world with 4000+ members. He is the initiator and key maintainer of the istio.io Chinese website.
```

**Review by linsun — APPROVED** [review]

**Pull request merged**